rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -1,5 +1,7 @@
 == Hardware Requirements
 
+* We recommend using a dedicated computer for running the RPA Bot.
```

Comment:
Our [style](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22MuleSoftrecommends%22(%22advises,%22%22suggests%22%E2%80%A6)) is to avoid "we recommend"